gwt-client-common/src/main/java/nl/aerius/wui/dev/GWTProd.java

```diff
@@ -71,23 +75,39 @@ public static void warn(final Object a, final Object b, final Object c) {
   }
 
   public static void error(final Object msg) {
-    DomGlobal.console.warn(msg);
+    if (dev) {
+      DomGlobal.console.error(msg);
+    } else {
+      LOGGER.severe(safeObjecString(msg));
+    }
     tryReport(msg);
   }
 
   public static void error(final Object a, final Object b) {
-    DomGlobal.console.error(a, b);
+    if (dev) {
+      DomGlobal.console.error(a, b);
+    } else {
+      LOGGER.severe(safeObjecString(a) + "-" + safeObjecString(b));
+    }
```

Comment:
Do we not always want to log to console, or does the LOGGER to do that as well?

Reply:
It should be possible to configure it to also log to console (in `gwt.xml`) (Except that when I try to enable it in my local build the compiler crashes...)

Reply:
Let's just do the `DomGlobal` one regardless of `dev` or not.. and use the `LOGGER` if it's not local development?